PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -812,6 +812,107 @@ BufferID FileManager::loadFile(const wchar_t* filename, Document doc, int encodi
 	}
 }
 
+bool FileManager::checkIfFileIsBinary(wstring fileExtension)
+{
+	for (size_t ii = 0; ii < _extensionsToNotCheckIfBinary.size(); ii++)
+	{
+		if (_extensionsToNotCheckIfBinary.at(ii) == fileExtension)
+			return false;
+	}
+	return true;
+}
+
+wstring FileManager::getExtensionsToNotCheckBinary()
+{
+	std::wstring joinedList;
+	stringJoin(_extensionsToNotCheckIfBinary, wstring(L"|"), joinedList);
+	return joinedList;
+}
+
+void FileManager::setExtensionsToNotCheckBinary(const TCHAR* joinedList)

```

Comment:
No more macros TCHAR & TEXT("") in the project. Use wchar_t & L"" respectively instead.